contrib/cirrus/runner.sh

```diff
@@ -52,19 +58,33 @@ _run_verify_vendor() {
     fi
 }
 
+_run_verify_vendor_aarch64() {
+    _run_verify_vendor
+}
+
 _run_unit() {
     make unit
 }
 
+_run_unit_aarch64() {
+    _run_unit
+}
+
 _run_integration() {
     make integration
 }
 
+_run_integration_aarch64() {
+    _run_integration
+}
+
 show_env_vars
 
 msg "************************************************************"
 msg "Toolchain details"
 msg "************************************************************"
+rustup install stable
+rustup default stable
```

Comment:
Minor nit: If these lines are going to stay for now, it might be nice to add a comment pointing maintainers at https://github.com/containers/automation_images/issues/150

Reply:
done. PTAL